reproduce-figures/README.md

```diff
@@ -1 +1,3 @@
 This directory contains instructions as well as necessary scripts and certain data files for reproducing manuscript figures and tables.
+
+* `prepare-metadata-files.R` is a script which can be used to generate metadata files needed to reproduce figures and tables, as described in `obtain-prepare-data.md`.
```

Comment:
I'm assuming this file is being added in the instructions PR?

Reply:
Yes, that's the main component to that PR!